specification/cognitiveservices/data-plane/AnomalyDetector/preview/v2.0-preview/RootCauseAnalysis.json

```diff
@@ -0,0 +1,2162 @@
+          }
+        }
+      }
+    },
+    "/rootCauseAnalysis/instances/{instanceId}/models/{modelId}/inferences/{inferenceId}": {
+      "get": {
+        "tags": [
+          "RootCauseAnalysisModel"
+        ],
+        "summary": "Query an inference task state for root cause analysis",
+        "operationId": "GetRootCauseAnalysisInferenceState",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "instanceId",
+            "description": "Instance unique id",
+            "required": true,
+            "type": "string",
+            "format": "uuid"
+          },
+          {
+            "in": "path",
+            "name": "modelId",
+            "description": "Model unique id",
+            "required": true,
+            "type": "string",
+            "format": "uuid"
+          },
+          {
+            "in": "path",
+            "name": "inferenceId",
+            "description": "Inference unique id",
+            "required": true,
+            "type": "string",
+            "format": "uuid"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/RootCauseAnalysisInferenceResponse"
+            }
+          },
+          "default": {
+            "description": "Client error or server error (4xx or 5xx)",
+            "schema": {
+              "$ref": "#/definitions/ErrorCode"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Query an inference task state for root cause analysis": {
+            "$ref": "./examples/GetRootCauseAnalysisInferenceState.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "RootCauseAnalysisMeasure": {
+      "required": [
+        "displayName"
+      ],
+      "type": "object",
+      "properties": {
+        "displayName": {
+          "description": "Root cause analysis measure name",
+          "type": "string"
+        }
+      }
+    },
+    "RootCauseAnalysisDataSet": {
+      "required": [
+        "displayName",
+        "kind"
+      ],
+      "type": "object",
+      "properties": {
+        "kind": {
+          "description": "Root cause analysis kind",
+          "enum": [
+            "multidimensional"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "RootCauseAnalysisType",
+            "modelAsString": true
+          }
+        },
+        "dataSetId": {
+          "format": "uuid",
+          "description": "Root cause analysis data set unique ID",
+          "type": "string",
+          "readOnly": true
+        },
+        "displayName": {
+          "description": "Root cause analysis data set name",
+          "type": "string"
+        }
+      },
+      "discriminator": "kind",
+      "example": {
+  "granularity": "daily",
+  "customInterval": 0,
```

Comment:
We use this customInterval to align with existing MVAD API set. It describe the unit of granularity. For example, when graularity is daily, customInterval is 5, it means  5 days granularity.